### ISSUE TITLE
Remove alpha tag for Script Link component

### ISSUE THREAD

**Issue by j-pendlebury** [BBC-archive/psammead#2256]
**Is your feature request related to a problem? Please describe.**
The script switch component is currently an alpha component.

This component should be amended and tested according to the a11y requirements to remove the tag.

**Describe the solution you'd like**
Remove alpha tag after successful a11y swarm on the component.

**Testing notes**
A11y testing.

Dev insight: Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**
WSFRONTPGS-100

**Comment by AlistairGempf** [comment]
Blocked on https://github.com/bbc/psammead/issues/2276

**Comment by Bopchy** [comment]
As the ScriptLink is only being used within the Brand component, the alpha tag can be removed from the ScriptLink component once the a11y swarm is complete

The issue of the SkipLink component being moved from Navigation to Brand will be handled in https://github.com/bbc/psammead/issues/2713
